src/bridge/RoomAccessSyncer.ts

```diff
@@ -88,9 +88,9 @@ export class RoomAccessSyncer {
     }
 
     public async removePowerLevels(roomId: string, users: string[], req: BridgeRequest) {
-        const map: {[userId: string]: null} = {};
-        users.forEach((u) => {map[u] = null});
-        return this.changePowerLevels(roomId, map, req);
+        return Promise.all(users.map(userId =>
```

Comment:
Avoids unexpected return types (array full of `void`/`undefined`).
```suggestion
        await Promise.all(users.map(userId =>
```

Also easier to read, because I won't be looking for a possible return type.